### PR TITLE
fix: added rbac guide for keto

### PR DESCRIPTION
**Pull request by gen1us2k**
## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [ ] I have added tests that prove my fix is effective or that my feature
      works.
- [ ] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: docs/keto/guides/rbac.mdx [hunk 1]
```diff
@@ -2,6 +2,234 @@
...
+      "object": "marketing",
+      "relation": "member",
+      "subject_id": "Dilan"
+    }
+  ],
+  "next_page_token": ""
+}
+
+# Get permissions to objects for marketing group
+curl -s http://localhost:4466/relation-tuples\?subject_set.namespace\=groups\&subject_set.relation\=member\&subject_set.object\=marketing | jq
```
Comment (hperl) [inline docs/keto/guides/rbac.mdx:202]: As above:
```bash
keto relation-tuple get --subject-set="groups:marketing#member"
```

File: docs/keto/guides/rbac.mdx [hunk 2]
```diff
@@ -2,6 +2,234 @@
...
+        "namespace": "groups",
+        "object": "marketing",
+        "relation": "member"
+      }
+    }
+  ],
+  "next_page_token": ""
+}
+# Get permissions to objects for community group
+curl -s http://localhost:4466/relation-tuples\?subject_set.namespace\=groups\&subject_set.relation\=member\&subject_set.object\=marketing | jq
```
Comment (hperl) [inline docs/keto/guides/rbac.mdx:219]: Here the request referenced "marketing", not "community".
```bash
keto relation-tuple get --subject-set="groups:community#member"
```

File: docs/keto/guides/rbac.mdx [hunk 3]
```diff
@@ -2,6 +2,234 @@
...
+**Benefits:**
+
+- Reduces management complexity when many identities share similar permissions.
+- Role hierarchies can reduce redundancy even further.
+- Is well established and easily understood by many developers as it is a
+  de-facto standard for web applications.
+
+**Shortcomings:**
+
+- Has no concept of context:
```
Comment (hperl) [inline docs/keto/guides/rbac.mdx:60]: It looks like the second half is missing here. I think we can just delete this line as you already explain "context" in the next bullet points, right?

**Review by hperl — CHANGES_REQUESTED** [review]

File: docs/keto/guides/rbac.mdx (hunk 2 above)
Comment (gen1us2k) [reply docs/keto/guides/rbac.mdx]: Thanks. I've fixed it.

File: docs/keto/guides/rbac.mdx (hunk 3 above)
Comment (gen1us2k) [reply docs/keto/guides/rbac.mdx]: right

**Comment by gen1us2k** [comment]
@hperl @zepatrik Hello. I fixed everything and it looks fine now. The guide uses `keto cli` instead of curl examples. WDYT?

**Review by hperl — APPROVED** [review]

**Comment by hperl** [comment]
> @hperl @zepatrik Hello. I fixed everything and it looks fine now. The guide uses `keto cli` instead of curl examples. WDYT?

Thanks, LGTM!

**Comment by hperl** [comment]
@zepatrik LGTM, but I can't approve. Can you?

**Pull request merged**